app/src/main/java/ncnf/feed/FeedActivity.java

```diff
@@ -0,0 +1,61 @@
+package ncnf.feed;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.ncnf.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ncnf.event.Event;
+import ncnf.event.EventType;
+import ncnf.event.PublicEvent;
+import ncnf.organizer.PublicOrganizer;
+
+public class FeedActivity extends AppCompatActivity implements EventAdapter.OnEventListener {
+    private RecyclerView.LayoutManager lManager;
+    private EventAdapter adapter;
+    private List<Event> eventList;
+    private static final String CHANNEL_NAME = "events_to_be_shown";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        //setContentView(R.layout.activity_feed);
+        setContentView(R.layout.activity_feed);
```

Comment:
We should maybe nest the component's views in a folder ? (e.g. layout/feed/main.xml)